tests/base_api_test.py

```diff
@@ -53,12 +53,14 @@ def _iterator(url, key):
         cls.saved_groups = _iterator('/v1/groups', 'groups')
         cls.saved_users = _iterator('/v1/users', 'users')
         cls.saved_roles = _iterator('/v1/roles', 'roles')
+        cls.saved_resources = _iterator('/v1/resource', 'resources')
```

Comment:
Should this be `/v1/resources`?

Reply:
never mind ([link](https://github.com/HumanCellAtlas/fusillade/blob/tsmith/resource-feat/fusillade-api.yml#L938))